app/src/sandbox/SandboxPage.tsx

```diff
@@ -36,6 +37,7 @@ export const SandboxPage = () => {
         { id: 'SkillsBatteryPopover', name: 'Skills', component: Skills },
         { id: 'TableCellsStylesSandbox', name: 'Table Cells/Rows styles', component: TableCellsStylesSandbox },
         { id: 'AdaptivePanel', name: 'Adaptive panel', component: AdaptivePanelDemo },
+        { id: 'testCX', name: 'Test inputCX & bodyCX', component: TestCX },
```

Comment:
let's remove this demo, i guess that it won't be useful in feature

Reply:
done